docs/src/pages/docs/operator-postgrespolicy.md

```
@@ -0,0 +1,178 @@
+    - role: legacy_app
+      reassign_owned_to: app_owner
+      drop_owned: true
+```
+
+Declared schemas can be created and have ownership converged by the operator. Schemas that are only referenced from top-level grants or default privileges must already exist in the database. Profile-generated roles also inherit the profile-level `login` and `inherit` attributes shown above.
+
+## Role passwords
+
+Roles can either sync a password from an existing Kubernetes Secret or ask the operator to generate and manage a per-role Secret. In both cases the password value is resolved at reconcile time and only sent to PostgreSQL inside the apply transaction.
+
+#### Sync from an existing Secret
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)